packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/simulationservice/SimulationCiemssServiceProxy.java

```diff
@@ -10,7 +10,7 @@
 import software.uncharted.terarium.hmiserver.models.simulationservice.JobResponse;
 
 
-@FeignClient(name = "ciemss-service", url = "${terarium.dataservice.url}", path = "/ciemss-service")
+@FeignClient(name = "ciemss-service", url = "${ciemss-service.url}", path = "")
```

Comment:
I believe here you can fully drop `path`

Reply:
done!